pages/database-management/authentication-and-authorization/users.mdx

```diff
@@ -51,6 +51,13 @@ log-in using any password:
 ```cypher
 SET PASSWORD FOR user_name TO null;
 ```
+
+User can change own password by running following command:
```

Comment:
Looks like the Grammarly check missed this

s/change own/change their own

Reply:
Thanks! Fixed.